### ISSUE TITLE
[front] style: making comparison on mobile devices less of a painful experience

### ISSUE THREAD

**Issue by GresilleSiffle** [tournesol-app/tournesol#1240]
### Context

The main problem identified is the scrolling behaviour. When a user tries to scroll the sliding movement will often be grabbed by the slider inputs instead of being interpreted as a page scroll. In those cases the slider handle will move, potentially provoking the loss of an already selected rating.

We may discover other challenges after having addressed this one.

### Proposition

This issue can be used to identify potential "quick wins" that will fix or at least greatly ease the comparison on mobile devices. 

Few random ideas:
- See if there are options in the MUI sliders documentation that can help us.
- Increase the gap between sliders on mobile, to increase the chance of touching the background instead of an input field (this will probably be not enough to fix the problem).
- Display a visible vertical scroll bar? (Just in case nothing else worked, we could precisely display the scroll-able surface).
- Any other ideas are welcome.

**Comment by sigmike** [comment]
> See if there are options in the MUI sliders documentation that can help us.

I didn't find any option that would help.

I found a possibly related npm package: https://www.npmjs.com/package/mui-scrollable-slider-hook. I tried it but it doesn't seem to solve the problem. It was made by the author of an anwser in https://stackoverflow.com/questions/64776560/scroll-events-unintentionally-changing-material-ui-slider-values.

I also found this issue that seems related: https://github.com/mui/material-ui/issues/20990. The behaviors described in [this comment](https://github.com/mui/material-ui/issues/20990#issuecomment-627246200) sound good, but it looks like they haven't been implemented yet.
